asserts/database.go

```diff
@@ -61,6 +60,20 @@ type Backstore interface {
 	SearchBySuffix(assertType AssertionType, primaryPathWithoutLast []string, suffixOflast string, foundCb func(Assertion)) error
 }
 
+// KeypairManager is a manager and backstore for private/public key pairs.
+type KeypairManager interface {
+	// Import stores the given private/public key pair for identity and
+	// returns its fingerprint
+	Import(authorityID string, privKey PrivateKey) (fingerprint string, err error)
+	// Get returns the private/public key pair with the given fingeprint.
+	Get(authorityID, fingeprint string) (PrivateKey, error)
```

Comment:
Given our conversation yesterday, should this take the key id instead, and hold the constraint of Find that if two keys have the same key id it fails?  We can then get rid of Find, I think.